db/schema.rb

```diff
@@ -33,6 +36,12 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
+  create_table "grades", force: :cascade do |t|
```

Comment:
I don't see migrations for this and `ratings` table. Are those some local leftovers?
Looks like you need to drop and recreate your local database.